core/core.go

```diff
@@ -44,12 +44,16 @@ type BlockLocation struct {
 type Block struct {
 	ID string `json:"id"`
 
-	// sha256 hash of the block
-	Sha256Hash string `json:"hash"`
+	// Offset of the block in the file, relative to the file's other blocks.
+	// For the first block, this is zero.
+	Num int `json:"blockNum"`
```

Comment:
Is this offset in bytes? Or is it a sequence? I assume its a sequence, but it might might be good to mention since it's a bit ambiguous.

Reply:
Sequence. You are right. I will update it.